fe/fe-core/src/main/java/org/apache/doris/qe/ConnectContext.java

```diff
@@ -450,6 +467,64 @@ public void setEnv(Env env) {
         defaultCatalog = env.getInternalCatalog().getName();
     }
 
+    public void setUserVar(SetVar setVar) {
+        userVars.put(setVar.getVariable().toLowerCase(), setVar.getResult());
+    }
+
+    public @Nullable Literal getLiteralForUserVar(String varName) {
```

Comment:
varName always lower case?

Reply:
user variable is case insensitive, so convert to lowercase